src/CMakeLists.txt

```diff
@@ -90,7 +90,18 @@ set(COMPILED_WITH ${CMAKE_CXX_COMPILER})
 set(COMPILER_COMMAND "<not-implemented>") # TODO: Put something more descriptive here
 set(COMPILER_FLAGS "<not-implemented>") # TODO: Put something more descriptive here
 
-set(COORDINATE_TYPE UniformCartesian) # TODO: Make this an option when more are available
+set(PARTHENON_COORDINATES "Cartesian" CACHE STRING "Set the coordinate system from {Cartesian, Cylindrical, Spherical} with Cartesian the default")
+
+if (${PARTHENON_COORDINATES} STREQUAL "Cartesian")
+  set(COORDINATE_TYPE UniformCartesian)
+elseif(${PARTHENON_COORDINATES} STREQUAL "Cylindrical")
+  set(COORDINATE_TYPE UniformCylindrical)
+elseif(${PARTHENON_COORDINATES} STREQUAL "Spherical")
+  set(COORDINATE_TYPE UniformSpherical)
+else()
```

Comment:
I don't love this switch statement honestly... I'd rather it be more flexible, but I won't push too hard on that.

Reply:
Dealt with this as you've already seen.